charts/osm/templates/fluentbit-configmap.yaml

```diff
@@ -17,10 +17,27 @@ data:
       Path    /var/log/containers/osm-controller-*_{{ .Release.Namespace }}_osm-controller-*.log
       Parser  cri
       Read_from_Head  on
+    # Helps grep filter identify logs generated by clusters running on cri-o
     [FILTER]
-      name       grep
-      match      *
-      regex       message /"level":"error"/
+      Name           modify
+      Match          kube.*
+      Condition      Key_value_matches message /"level":"{{ .Values.OpenServiceMesh.fluentBit.logLevel }}"/
+      Set            keep true
+    # Helps grep filter identify logs generated by clusters running on moby, containerd
+    [FILTER]
+      Name           modify
+      Match          kube.*
+      Condition      Key_value_matches log \\"level\\":\\"{{ .Values.OpenServiceMesh.fluentBit.logLevel }}\\"
+      Set            keep true
+    [FILTER]
+      Name           grep
```

Comment:
can you add a comment for this filter?

Reply:
I've updated all the comments in my most recent commit, let me know if they seem clearer now